VictoryCenter/VictoryCenter.WebAPI/.env.example

```
@@ -8,3 +8,7 @@ WAY4PAY_MERCHANT_LOGIN=t_me_6ca2e
 WAY4PAY_MERCHANT_SECRET_KEY=32555490f7f4328b45e437834639404b02525558
 WAY4PAY_MERCHANT_DOMAIN_NAME=https://example.com
 WAY4PAY_API_URL=https://secure.wayforpay.com/pay
+CLOUDFLARE_TURNSTILE_SECRET_KEY=your-cloudflare-turnstile-secret-key
+CLOUDFLARE_TURNSTILE_SITE_VERIFY_URL=https://challenges.cloudflare.com/turnstile/v0/siteverify
+CONTACT_US_EMAIL_OPTIONS_FROM_ADDRESS=your_email_address@example.com
+RESEND_API_TOKEN=your_resend_api_token
```

dotenv-linter (4.0.0)

[warning] [EndingBlankLine] No blank line at the end of the file

(EndingBlankLine)